New version: Sparlectra v0.4.10

JuliaRegistrator opened the pull request:
- Registering package: Sparlectra
- Repository: https://github.com/Welthulk/Sparlectra.jl
- Created by: @Welthulk
- Version: v0.4.10
- Commit: 715db7c0f56e1e748b7dbc86485360cdffd7e533
- Reviewed by: @Welthulk
- Reference: https://github.com/Welthulk/Sparlectra.jl/issues/25#issuecomment-2031832339
- Description: load flow calculation using newton-raphson
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598c824e244585aa99d1df8eecb79c34799785b021949271e085f610ae5ccbc3ef80fd2df7f0226e46c884fb03d2b6395877cc52acc91d921c106e69cfc644fec7d7f45d184eec9eecfebf0dd2e31538812b5b3ffdcdb69748159e377f6d0d5fe9d67e45862dca8460eba6e29aed0c924ecf2553004e41def1289039e759efb96ca -->

Welthulk commented:
merge approved

Welthulk commented:
[merge approved]

Welthulk commented:
why is my new release not published?

GunnarFarneback commented:
[noblock]

You still need to add `[noblock]` to your comments to unblock this PR. (At least to the first one, unsure about the second.)

JuliaRegistrator committed:
New version: Sparlectra v0.4.10

UUID: 31ce9bba-fd9d-44a1-b005-f5f509afda38
Repo: https://github.com/Welthulk/Sparlectra.jl.git
Tree: 35421a00f0d3ba44f8456c6f5c00b8243a9a419d

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
